lib/src/common/repos_service.dart

```diff
@@ -208,86 +247,192 @@ class RepositoriesService extends Service {
         (json) => Collaborator.fromJson(json),
       );
 
-  Future<bool> isCollaborator(RepositorySlug slug, String user) {
-    return _github
-        .request("GET", "/repos/${slug.fullName}/collaborators/$user")
-        .then((response) {
-      return response.statusCode == 204;
-    });
+  Future<void> isCollaborator(RepositorySlug slug, String user) async {
+    return _github.request(
+      "GET",
+      "/repos/${slug.fullName}/collaborators/$user",
+      statusCode: 204,
+    );
   }
 
-  Future<bool> addCollaborator(RepositorySlug slug, String user) {
-    return _github
-        .request("PUT", "/repos/${slug.fullName}/collaborators/$user")
-        .then((response) {
-      return response.statusCode == 204;
-    });
+  Future<void> addCollaborator(RepositorySlug slug, String user) async {
+    return _github.request(
+      "PUT",
+      "/repos/${slug.fullName}/collaborators/$user",
+      statusCode: 204,
+    );
+  }
+
+  Future<void> removeCollaborator(RepositorySlug slug, String user) async {
+    return _github.request(
+      "DELETE",
+      "/repos/${slug.fullName}/collaborators/$user",
+      statusCode: 204,
+    );
+  }
+
+  /// Returns a list of all comments for a specific commit.
+  ///
+  /// https://developer.github.com/v3/repos/comments/#list-commit-comments-for-a-repository
+  Stream<CommitComment> listComments(
```

Comment:
Oh, wait, maybe this method should be `listCommitComments` and the other one `listComments` ?

Reply:
I renamed it `listSingleCommitComments`, what do you think?